parser-typechecker/tests/Unison/Test/UnisonSources.hs

```diff
@@ -60,14 +60,14 @@ bad r = EasyTest.expectLeft r >> done
 
 test :: Test ()
 test = do
-  rt <- io (RTI.startRuntime False RTI.OneOff "")
-  scope "unison-src"
-    . tests
-    $ [ go rt shouldPassNow good,
-        go rt shouldFailNow bad,
-        go rt shouldPassLater (pending . bad),
-        go rt shouldFailLater (pending . good)
-      ]
+  using (RTI.startRuntime False RTI.OneOff "") RTI.terminate \rt -> do
```

Comment:
Was there some corresponding to `RTI.terminate` that we need to delete?

Reply:
Oops, disregard, I'm only viewing one commit of this PR, somehow. And now I see that this is just a test file.